Source/WebCore/page/ContextMenuController.cpp

```diff
@@ -1565,6 +1583,18 @@ void ContextMenuController::showImageControlsMenu(Event& event)
 
 #endif
 
+
+#if ENABLE(PDFJS)
+
+void ContextMenuController::performPDFJSAction(Frame* frame, const String& action)
```

Comment:
This should take a reference to `Frame` – there is a null check at the top of `ContextMenuController::contextMenuItemSelected`.

Reply:
Done, thanks!